aws/logs_monitoring/enhanced_lambda_metrics.py

```diff
@@ -1,16 +1,22 @@
 import logging
 import os
 import re
+import json
+import datetime
 
 from collections import defaultdict
 from time import time
 
 import boto3
 from botocore.exceptions import ClientError
 
-ENHANCED_METRICS_NAMESPACE_PREFIX = "aws.lambda.enhanced"
+# from settings import DD_S3_BUCKET_NAME, DD_S3_CACHE_FILENAME, DD_TAGS_CACHE_TTL_SECONDS
+# TODO: the above line causes an issue with the unit tests
```

Comment:
Per Slack convo -- I think this issue can be fixed by patching the DD_API_KEY env var.